homeassistant/components/jellyfin/strings.json

```diff
@@ -1,7 +1,16 @@
 {
   "config": {
     "step": {
+      "reauth_confirm": {
+        "title": "[%key:common::config_flow::title::reauth%]",
+        "description": "The Jellyfin integration needs to re-authenticate your account",
+        "data": {
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      },
       "user": {
+        "title": "Login",
+        "description": "Enter your Jellyfin URL, username and password to login",
```

Comment:
```suggestion
```
Don't think this adds anything right?

Reply:
I suppose we can indeed do without. I removed these lines.